src/app/app.component.html

```diff
@@ -1,9 +1,44 @@
 <p-growl [(value)]="messages" life="3000"></p-growl>
 
-<gb-main *ngIf="authenticationCompleted"></gb-main>
+<gb-main *ngIf="authenticatedAndServerUp"></gb-main>
 
-<div class="gb-page" *ngIf="!authenticationCompleted">
-  <div class="gb-message-box gb-info">
-  You are being redirected to the login page to authenticate ...
+<div *ngIf="!authenticatedAndServerUp">
+
+  <div class="gb-title">
+    Glowing Bear
+    <a *ngIf="docUrl" [href]="docUrl" target="_blank"><small>{{appVersion}}</small>&nbsp; <i class="fa fa-question-circle-o"></i></a>
+    <span *ngIf="!docUrl"><small>{{appVersion}}</small></span>
   </div>
+
+  <div id="statusMessages">
+
+    <div class="gb-page" *ngIf="configLoaded && !authenticationCompleted">
+      <div class="gb-message-box gb-info">
+      You are being redirected to the login page to authenticate ...
+      </div>
+    </div>
+
+    <div class="gb-page" *ngIf="!configLoaded">
+      <div class="gb-message-box gb-error">
+        <p>The configuration is not loaded correctly</p>
+        <code *ngIf="configError">
+          {{configError}}
+        </code>
+      </div>
+    </div>
+
+    <div class="gb-page" *ngIf="serverDown">
+      <div class="gb-message-box gb-error">
+        The server is down.
```

Comment:
Should it be only for transmart? Or gb-backend as well? It also has the availability endpoint.

Reply:
Indeed, that would be a great addition. I'll create an issue for that.